BikeRental.tests/RentalTests.cs

```diff
@@ -0,0 +1,99 @@
+using BikeRental.Domain;
+using Xunit;
+using System.Collections.Generic;
+using System.Linq;
+using BikeRental.Domain.Models;
+using BikeRental.Domain.Enums;
+
+namespace BikeRental.Tests;
+
+public class RentalTests
+{
+    private readonly List<BikeModel> _models;
+    private readonly List<Bike> _bikes;
+    private readonly List<Renter> _renters;
+    private readonly List<Rental> _rentals;
+
+    public RentalTests()
+    {
+        _models = DataSeeder.GetBikeModels();
+        _renters = DataSeeder.GetRenters();
+        _bikes = DataSeeder.GetBikes(_models);
+        _rentals = DataSeeder.GetRentals(_bikes, _renters);
+    }
+
+    [Fact]
+    public void Should_Find_All_Sport_Bikes()
+    {
+        var sportBikes = _models.Where(m => m.Type == BikeType.Sport).ToList();
+        Assert.NotEmpty(sportBikes);
+    }
+
+    [Fact]
+    public void Should_Calculate_Top5_Models_By_Profit()
+    {
+        var top5 = _rentals
+            .GroupBy(r => r.Bike.Model)
+            .Select(g => new
+            {
+                Model = g.Key,
```

Comment:
Я бы предложил в результат брать не всю информацию о модели а только наименование.
```suggestion
                Model = g.Key.Name,
```